### PR TITLE
Add 4.7.0 ❄️ 

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Upstream: https://github.com/maxmind/GeoIP2-python/tree/v4.7.0

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+channels:
+  - sfe1ed40
+
+upload_channels:
+  - sfe1ed40
```
Comment (lorepirri) [inline abs.yaml]:
```suggestion
upload_channels:
  - sfe1ed40
```

Comment (lorepirri) [reply abs.yaml]: https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Building_Snowflake_packages.md#tldr

**Review by lorepirri — APPROVED** [review]
you can remove the `channels` in `abs.yaml`. Only `upload_channels` will be used.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,32 +10,46 @@ source:
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
+    - setuptools-scm
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
I don't see it being used in this version.

**Review by cbouss — APPROVED** [review]
Seems to be that you don't need setuptools-scm, but overall LGTM.

**Pull request merged**